From 9345fd9949ba0f8c8711e0f5f4ad9ed4e004f8a9 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Mon, 18 Feb 2013 16:37:21 +0000
Subject: [PATCH 0326/1000] LogService.exe massaiSNMP.dll V1.0.4.5 Mantis
 0025834: BSS: new component for BaggageSortingSystem new
 event-/comp-configuration for BaggageSortingSystem

git-svn-id: svn://localhost/SelfServiceCommon/trunk@490 90b65887-3827-0410-9a23-83215b262276
---
 .../compSource/BaggageSortingSystem.GENERAL.compDefinition.cfg  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/BaggageSortingSystem.GENERAL.compDefinition.cfg b/SelfServiceCommon/Massai/cpp/LogService/compSource/BaggageSortingSystem.GENERAL.compDefinition.cfg
index 8eed756a..b9b67a0e 100644
--- a/SelfServiceCommon/Massai/cpp/LogService/compSource/BaggageSortingSystem.GENERAL.compDefinition.cfg
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/BaggageSortingSystem.GENERAL.compDefinition.cfg
@@ -1,3 +1,3 @@
-BSS_OUTPUT;OK=2070;ERROR=2072 | 2073 | 2074 | 2075
+BSS_OUTPUT;OK=2080;ERROR=2082 | 2083 | 2084 | 2085
 BSS_COMP;OK=BSS_OUTPUT.OK
 CUSS_BSS_COMP(HIDDEN);OK=BSS_OUTPUT.OK
\ No newline at end of file
-- 
2.41.0.windows.1

